cluster-autoscaler/utils/nodegroupset/compare_nodegroups.go

```diff
@@ -118,10 +127,19 @@ func IsNodeInfoSimilar(n1, n2 *schedulercache.NodeInfo) bool {
 			if label == kubeletapis.LabelZoneRegion {
 				continue
 			}
-			if label == kubeletapis.LabelGardenerWorkerGroup {
+			if label == LabelGardenerWorkerGroup {
```

Comment:
Why this change? If we change this, we could change `kubeletapis.LabelZoneRegion` also or keep both?

Reply:
Please see the comment above.